README.md

```diff
@@ -1,4 +1,7 @@
 <<<<<<< HEAD
+
+Hello world here
+
```

Comment:
This code looks good

---
**Sema Reaction:** :ok_hand: Looks good

Reply:
Please revise this code

---
**Sema Reaction:** :hammer_and_wrench: Fix

Reply:
Code does not make sense

---
**Sema Reaction:** :hammer_and_wrench: Fix | **Sema Tags:** Unreadable, Not secure, Inefficient, Not reusable